loopy/target/pyopencl.py

```diff
@@ -110,6 +117,28 @@ def with_types(self, arg_id_to_dtype, callables_table):
                 self.copy(arg_id_to_dtype=arg_id_to_dtype),
                 callables_table)
 
+    def generate_preambles(self, target):
+        name = self.name_in_target
+        if name.startswith("lpy_real") or name.startswith("lpy_conj"):
+            dtype = self.arg_id_to_dtype[-1]
+            ctype = target.dtype_to_typename(dtype)
+
+            yield(f"40_{name}", f"""
```

Comment:
What's the number in front supposed to mean? I just copied this from the [c target](https://github.com/inducer/loopy/blob/7c90b7e3134467d20d1d8182960e965783f74499/loopy/target/c/__init__.py#L642-L658).

Reply:
Those are just some identifiers to act as sorting keys determining their location in the generated code. I think this is fine.

Reply:
Ah, I see. Thanks for all the help with this!